tools/scripts/maxim.mk

```diff
@@ -3,7 +3,7 @@ $(error MAXIM_LIBRARIES not defined.$(ENDL))
 endif
 
 UNIX_TOOLS_PATH := $(MAXIM_LIBRARIES)/../Tools/MinGW/msys/1.0/bin
-ARM_COMPILER_PATH := $(MAXIM_LIBRARIES)/../Tools/GNUTools/bin
+ARM_COMPILER_PATH := $(MAXIM_LIBRARIES)/../Tools/GNUTools/10.3/bin
```

Comment:
This is going to keep changing in the future.
How about you do something like this:
https://github.com/analogdevicesinc/no-OS/blob/master/tools/scripts/stm32.mk#L27-L28

Basically you search for a path with a certain pattern (like *bin/arm-none-eabi-gcc) and then resolve just the /full/path/to/bin using Make libraries functions and avoid having to maintain this file

Reply:
I'm actually now wondering how this explicit version in makefile would work for users who have different versions installed